wom/src/main/scala/wom/graph/CallNode.scala

```diff
@@ -25,6 +25,9 @@ sealed abstract class CallNode extends GraphNode {
   def callType: String
 
   def inputDefinitionMappings: InputDefinitionMappings
+  def mustFollow: Set[GraphNode]
```

Comment:
I tried to avoid using a generic name like `upstream` because this is not a complete set, it's *just* the `after`s (eg to get a complete set of upstreams you add "input upstreams" and "non-input upstreams".

But I agree this is ambiguous - I'll change it while trying to preserve that distinction.